cmd/ical-relay/config.yml.example

```diff
@@ -1,9 +1,11 @@
-version: 2
+version: 4
```

Comment:
do we need to go from version 2 to 4?

Reply:
yes, there was a version 3 in between, which was not updated in the example.